File: app/src/main/java/com/izettle/localconfig/application/database/tables/ApplicationTable.java
package com.izettle.localconfig.application.database.tables;

import com.izettle.localconfig.application.library.ApplicationCursorParser;

public class ApplicationTable {
    public static final String TABLE_NAME = "application";

    public static final String CREATE = "CREATE TABLE " + TABLE_NAME
            + "("
            + ApplicationCursorParser.Columns._ID + " INTEGER PRIMARY KEY, "
            + ApplicationCursorParser.Columns.APPLICATION_NAME + " TEXT "
            + ")";

}

File: wrench-app/src/main/java/com/izettle/wrench/database/tables/ApplicationTable.kt
package com.izettle.wrench.database.tables

interface ApplicationTable {
    companion object {
        const val TABLE_NAME = "application"
        const val COL_ID = "id"
        const val COL_PACK_NAME = "packageName"
        const val COL_APP_LABEL = "applicationLabel"
    }
}
